tests/sentry/incidents/endpoints/test_project_alert_rule_details.py

```diff
@@ -120,18 +100,7 @@ def test_simple(self):
             resp = self.get_success_response(
                 self.organization.slug, self.project.slug, self.alert_rule.id
             )
-
-        assert resp.data == serialize(self.alert_rule)
-
-    def test_aggregate_translation(self):
-        self.login_as(self.owner_user)
-        alert_rule = self.create_alert_rule(aggregate="count_unique(tags[sentry:user])")
-        with self.feature("organizations:incidents"):
-            resp = self.get_success_response(
-                self.organization.slug, self.project.slug, alert_rule.id
-            )
-            assert resp.data["aggregate"] == "count_unique(user)"
-            assert alert_rule.snuba_query.aggregate == "count_unique(tags[sentry:user])"
+        assert resp.data == serialize(self.alert_rule, serializer=DetailedAlertRuleSerializer())
```

Comment:
The `DetailedAlertRuleSerializer` subclasses the `AlertRuleSerializer` (which the `ProjectAlertRuleDetails` endpoint was using) and adds the following fields to the response (obviously with varying data):

`excludedProjects': [], 'eventTypes': ['error'], 'snooze': False`